### PR TITLE
Created Korean Translations 

### PR DESCRIPTION
**Pull request by rafa-gould**
Will fail because of a lack of unicode support, however when unicode support is added it will work!

Thank you so much!

### CODE REVIEW

**Comment by kattni** [comment]
Thanks so much! We're going to leave this open until we can support it.

File: locale/ko.po [hunk 1]
```diff
@@ -0,0 +1,2570 @@
...
+msgid "Failed to allocate RX buffer of %d bytes"
+msgstr ""
+
+#: ports/nrf/common-hal/bleio/Adapter.c
+msgid "Failed to change softdevice state"
+msgstr ""
+
+#: ports/nrf/common-hal/bleio/Device.c
+msgid "Failed to connect:"
+msgstr 연결할 수 없다""
```
Comment (jepler) [inline locale/ko.po]: Looks like there is a punctuation problem on this line, which may be the reason the build failed.

Comment (tannewt) [reply locale/ko.po]: @jepler I fixed it to see what happens now. Feel free to fix the file through github like I did. I think you have permissions too.

**Comment by jepler** [comment]
With #2042 in the mix, this DOES build a final .uf2, though 134 characters are missing from the bitmap font.  Building circuitplayground_express, there are now `3936 bytes free in flash out of 253440 bytes ( 247.5 kb ).`

**Comment by jepler** [comment]
@tannewt how to get this to re-try travis ci with the new commits on master for #2042?  I re-ran the build after you merged 2042, but it failed in the same way which is unexpected...

**Comment by tannewt** [comment]
@jepler We'll need to either rebase or merge it in. (I'm sure we'll need to rerun make translate too.) We both should be able to push to the original branch from our local git. If we can't, we can always make a new branch.

**Comment by jepler** [comment]
Testing performed: built locally for metro m4 express, connected from unicode-capable terminal program, provoked an error:
image
There are still unavailable characters in the on-board font.  I don't know if this should be enough to prevent inclusion of this patch.

**Comment by jepler** [comment]
@tannewt can we get a fresh review from you?   I think maybe this can go in, unless the missing font characters must be considered a blocker.

**Comment by dhalbert** [comment]
We did consider the lack of glyphs to be a blocker, and adding glyphs could easily overflow the small builds. So I think we will continue to hold this open.

**Comment by jepler** [comment]
Can we break this down into additional issues? Maybe:
 * Double-wide glyphs support (in memory format & rendering)
 * Actual Hangul glyph availability
 * Enabling only on larger devices, as the storage for the font will be large

**Comment by jepler** [comment]
@rafa_gould thanks for your work in this PR, I'm sorry technical limitations are making us slow to accept it.

**Comment by tannewt** [comment]
I'd rather merge this in and be ok with missing glyphs in the terminal. I was thinking we could later turn the terminal off for fonts that are too big to store (chinese). For Korean, we could probably just add the missing glyphs later.

**Comment by jepler** [comment]
@tannewt I think we could do so if you give us an approving review.  I'm loathe to do it since I tinkered so much in this PR.

**Pull request merged**